packages/plugin-core/src/_extension.ts

```diff
@@ -796,25 +813,51 @@ async function showWelcomeOrWhatsNew({
         previousExtensionVersion,
       });
       await StateService.instance().setGlobalVersion(version);
+
+      // ^t6dxodie048o
+      const toastOrView = UPGRADE_TOAST_OR_VIEW_TEST.getUserGroup(
```

Comment:
doesn't have to be in this pr but moving forward, would be good to encapsulate this in a funciton

```ts
ACTIVE_TESTS: [key: AbTestEnumKey]: AbTest = {}

treatment = getTreatment<T extends AbTest>(testname: AbTestEnumKey): T
```